inlong-sdk/sort-sdk/src/test/java/org/apache/inlong/sdk/sort/impl/InLongTopicManagerImplTest.java

```diff
@@ -63,7 +62,9 @@ public void setUp() throws Exception {
         SortClientConfig sortClientConfig = PowerMockito.mock(SortClientConfig.class);
         when(clientContext.getConfig()).thenReturn(sortClientConfig);
         when(sortClientConfig.getSortTaskId()).thenReturn("test");
+        when(sortClientConfig.getUpdateMetaDataIntervalSec()).thenReturn(60);
         queryConsumeConfig = PowerMockito.mock(QueryConsumeConfigImpl.class);
+        inLongTopicManager = new InLongTopicManagerImpl(clientContext, queryConsumeConfig);
```

Comment:
Miss the mock operation of queryConsumeConfig instance.

Reply:
This parameter needs to be instantiated, and the following test cases need to use it